primitive/src/main/java/io/atomix/primitive/partition/impl/PrimaryElectorService.java

```diff
@@ -310,10 +311,19 @@ ElectionState cleanup(Session session) {
                 .collect(Collectors.toList());
         if (primary.sessionId() == session.sessionId().id()) {
           if (!updatedRegistrations.isEmpty()) {
+            // find 1st backup to use as primary
+            Registration newPrimary = updatedRegistrations.get(0);
+            List<GroupMember> backups = term().backups(1);
+            if (!backups.isEmpty())
+            {
```

Comment:
Code style - brackets on same line as condition

Reply:
Ok, will fix and update the PR